### PR TITLE
Add GoogleApiException based on FacebookApiException

### PR DESCRIPTION
**Pull request by zakrava**
It serves better Exception.

### CODE REVIEW

File: src/Kdyby/Google/exceptions.php [hunk 1]
```diff
@@ -22,6 +22,111 @@
...
+	 * Make a new API Exception with the given result.
+	 *
+	 * @param array $result The result from the API server
+	 */
+	public function __construct($result)
+	{
+		$this->result = $result;
+
+		$code = 0;
+		if (isset($result['error_code']) && is_int($result['error_code'])) {
```
Comment (fprochazka) [inline src/Kdyby/Google/exceptions.php]: I'm confused, did you just copy the exception from facebook and renamed it without actually using it anywhere?

Comment (zakrava) [inline src/Kdyby/Google/exceptions.php]: I used it in Login example to catch exception instead of \Exception

Comment (fprochazka) [inline src/Kdyby/Google/exceptions.php]: But that doesn't make any sense...? Why would you catch this exception, if there is nothing that throws it?

**Comment by fprochazka** [comment]
I'm sorry, but this change alone has no added value.

**Pull request closed without merging**